tests/bootstrap_tests/test_configuration_writer.py

```diff
@@ -293,7 +293,7 @@ def _create_test_data(self, create_project):
             self.__project = self.mockgun.create(
                 "Project",
                 {
-                    "code": "TestWritePipelineConfigFile",
+                    "name": "TestWritePipelineConfigFile",
```

Comment:
Bad test data. The name of a project is in the `name` field. More on why this change was introduced bellow.